CMakeLists.txt

```diff
@@ -26,7 +26,10 @@ else()
 endif(FLTK_FOUND)
 
 if(MSVC)
-    find_package(LIBLO)
+    find_package(liblo)
```

Comment:
What happened here? Did `find_package(LIBLO)` not work?

Reply:
That one points to our custom findliblo, which was renoved. The one i added points to findliblo which is added within liblo's cmake integration.

Reply:
Oh, so `cmake`'s integration requires it in small letters?

Reply:
Basically yeah.